docs/admin/install.mdx

```diff
@@ -8,28 +8,13 @@ slug: /admin/installation
 ---
 
 Until the main Kwil DB repository is open source, the `kwil-admin` command line
-application may be (a) used from within the Kwil Docker image or (b) downloaded
-from the `kwil-cli` binary release repository.
+application may be downloaded from the Kwil binary release repository.
 
-## From within the Kwil Docker Image
 
-To install the most recent Kwil image, run:
+## Download the `kwil-admin` Executable
 
-```bash
-docker pull kwildb/kwil:latest
-```
-
-With a running container (e.g. `docker container run kwild:latest`), open an
-interactive shell in the container where you can run `kwil-admin` commands:
-
-```bash
-docker exec -it <container-name-or-id> /bin/sh
-```
-
-## Download the Standalone `kwil-admin` Executable
-
-The Kwil daemon (`kwild`) and the node administration tool will be
-packaged together. Download the [latest release](https://github.com/kwilteam/binary-releases/releases/tag/v0.6.0-beta)
+The Kwil admin tool(`kwil-admin`) will be packaged together with Kwil daemon (`kwild`) and  Kwil cli(`kwil-cli`). 
+Download the [latest release](https://github.com/kwilteam/binary-releases/releases/tag/v0.6.0-beta)
```

Comment:
We *could* do "https://github.com/kwilteam/binary-releases/releases/latest" with "latest" so we don't have to update docs as frequently.

Reply:
Yeah I want to change to latest too, should we do it?

Reply:
agree latest would be better